gnomad_qc/v4/analyses/grpmax_comps.py

```diff
@@ -0,0 +1,345 @@
+    pprint(results_dict, sort_dicts=False)
+    return results_dict
+
+
+def create_table(
+    version_dict: Dict[str, Dict[str, Dict[str, int]]],
+    data_subset: str = None,
+    grpmax_counts: bool = False,
+    non_syn_only: bool = False,
+):
+    """
+    Create tables of grpmax stats.
+
+    Tables have a column for each version as well as the difference for each grpmax group.
+
+    :param version_dict: Dictionary of grpmax stats
+    :param data_subset: Data subset
+    :param grpmax_counts: Results calculated using only grpmax counts for diverse genetic ancestry groups
+    """
+    headers = ["genetic ancestry group", "v2", "v4", "difference(v4-v2)"]
+    v2_dict_index = (
+        version_dict["v2"][data_subset] if grpmax_counts else version_dict["v2"]
+    )
+    v4_dict_index = (
+        version_dict["v4"][data_subset] if grpmax_counts else version_dict["v4"]
+    )
+    grps = deepcopy(DIVERSE_GRPS)
+
+    if grpmax_counts:
+        grps.append("all")
+    else:
+        grps.append("nfe")
+
+    for threshold in AF_THRESHOLDS:
+        table = []
+        for grp in grps:
+            v2_val = v2_dict_index[threshold].get(grp, 0)
+            v4_val = v4_dict_index[threshold].get(grp, 0)
+            diff = v4_val - v2_val
+            if diff < 0:
+                diff = f"\033[91m{diff}\033[0m"
+            else:
+                diff = f"\033[92m{diff}\033[0m"
+            table.append([grp, v2_val, v4_val, diff])
+        logger.info(
+            f"\n{'non-synonymous' if non_syn_only else ''} variant count by genetic "
+            f"ancestry group where the\n{'grpmax' if grpmax_counts else ''} AF is above"
+            f" {threshold}{' and the ' + str(data_subset) + ' AF is below it' if grpmax_counts else ''}...\n"
+            f"{tabulate(table, headers=headers, tablefmt='fancy_grid')}"
+        )
+
+
+def main(args):
+    """Find grpmax stats for gnomAD v4 and v2."""
+    version_dict = {}
+    grpmax_counts = args.grpmax_counts
+    non_syn_only = args.non_syn_only
+
+    for version in ["v4", "v2"]:
+        if version == "v4":
+            ht = release_sites().ht()
+        else:
+            ht = get_gnomad_public_data("exomes")
+        if args.test:
+            # Filter to two partitions
+            ht = ht._filter_partitions([0, 1])
+        t_variants = ht.count()
+
+        logger.info(
+            "Total number of variants in %s before filtering: %s", version, t_variants
+        )
+        msg = ""
+        # All MANE Select transcripts are canonical
+        if args.canonical:
+            logger.info("Filtering to only MANE Select and canonical transcripts...")
+            ht = filter_vep_to_canonical_transcripts(ht, filter_empty_csq=True)
+            msg = "canonical transcript filtering"
+
+        if non_syn_only:
+            logger.info("Filtering to keep only non-synonymous variants...")
+            # NOTE: There is no garauntree the most severe consequence is from the
```

Comment:
```suggestion
            # NOTE: There is no guarantee the most severe consequence is from the
```

Reply:
though (with my other comment) I guess you could run [this](https://github.com/broadinstitute/gnomad_methods/blob/main/gnomad/utils/vep.py#L830) to avoid needing this comment